src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -1,47 +1,51 @@
 /** @jsx React.DOM */
 
+"use strict";
 
 var React = require("react/addons");
 
-module.exports = React.createClass({
```

Comment:
meaning `module.exports = ` instead of `var LOL = ...` and `module.exports = LOL;`?